shared/trainer/defaults.yaml

```diff
@@ -118,6 +119,8 @@ dis:
     init_type: kaiming
     init_gain: 0.2
     use_sigmoid: false
+    num_D: 1 #Number of discriminators to use (>1 means multi-scale)
```

Comment:
don't we want `3` as default? is 1 enough?

Reply:
Sure, I just didn't want to make the choice on everyone's behalf ;) I'll change it